src/dispatching/repls/dialogues_repl.rs

```diff
@@ -30,7 +30,7 @@ where
     D: Clone + Default + Send + 'static,
     Fut: Future<Output = DialogueStage<D>> + Send + 'static,
     R: Requester + Send + Clone + 'static,
-    <R as Requester>::GetUpdates: Send,
+    <R as Requester>::GetUpdatesFaultTolerant: Send,
```

Comment:
Something is not right here, we've deleted `GetUpdatesFaultTolerant` a while ago 🤔

Reply:
Well because `dialogues_repl.rs` is simply unused. I've just removed it according to your suggestion.